apps/explorer/utils/__init__.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas
+import yaml
+import zipfile
+
+from io import BytesIO, StringIO
+
+
+def export_pixelsets(pixel_sets):
```

Comment:
Can you add a docstring here (using [NumPy Style](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html))?